src/udx.c

```diff
@@ -788,7 +788,8 @@ process_packet (udx_socket_t *socket, char *buf, ssize_t buf_len, struct sockadd
 
 static void
 remove_next (udx_fifo_t *f) {
-  while (f->len > 0 && udx__fifo_shift(f) == NULL);
+  while (f->len > 0 && udx__fifo_shift(f) == NULL)
+    ;
```

Comment:
ClangFormat change.

Reply:
How do I test/apply that myself in the future?

Reply:
Either using the CLI (https://clang.llvm.org/docs/ClangFormat.html#standalone-tool, `brew install clang-format`) or using an editor integration.

Reply:
👍